v2/thirdpartypasswordless/migration/from-auth0/account-migration/modifications-to-login.mdx

```diff
@@ -0,0 +1,528 @@
+---
+id: modifications-to-login
+title: Modifications to login
+hide_title: true
+---
+import TabItem from '@theme/TabItem';
```

Comment:
For now, you can make this file empty and add a placeholder saying coming soon. This is something that @jscyo will work on after.
Instead of removing all the content here, just comment it out.

Reply:
I updated the content of the about page and removed the rest from the sidebar.